src/atoms/link.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Link from '@mui/material/Link';
+
+interface UnderlineLinkProps {
+    children: string;
+    linkType: 'primary' | 'secondary' | 'default';
+    underline: 'none' | 'hover' | 'always';
+}
+
+function UnderlineLink({ children, linkType, underline }: UnderlineLinkProps) {
```

Comment:
"none" could be the default value for underline, if no value is provided.
component name should be Link.

Reply:
esta bien ingeniero, voy a poner que si no pasan underline quede por defecto el 'none'